.circleci/config.yml

```diff
@@ -48,6 +21,37 @@ templates:
         - run:
             name: Upload code coverage report
             command: bash <(curl -s https://codecov.io/bash)
+    job: &job-template
```

Comment:
Not important, but this labelling of templates confused me at first, I was looking for places where `job` was used. In the Travis jobs I used just a list, IMHO is a bit less noisy:
```yaml
templates:
    - &test-template
        docker: ...
    - &job-template
        docker: ...
```

Reply:
Nice, it did not come to my mind this is possible